src/Microsoft.Azure.SignalR.Common/ClientInvocation/IClientResultsManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IClientResultsManager
+    {
+        string GetNewInvocationId(string connectionId, string serverGUID);
+
+        void AddServiceMappingMessage(string invocationId, ServiceMappingMessage serviceMappingMessage);
```

Comment:
Redundant parameters. Either `AddServiceMapping(string invocationId, string instanceId)` or `AddServiceMapping(ServiceMappingMessage message)` is enough.

Reply:
fixed